src/mds/Server.cc

```diff
@@ -1132,12 +1132,10 @@ void Server::find_idle_sessions()
     return;
   }
 
+  std::vector<Session*> to_evict;
+
   bool defer_session_stale = g_conf().get_val<bool>("mds_defer_session_stale");
   const auto sessions_p1 = mds->sessionmap.by_state.find(Session::STATE_OPEN);
-  bool defer_client_eviction =
-  g_conf().get_val<bool>("defer_client_eviction_on_laggy_osds")
```

Comment:
Since the `defer_client_eviction_on_laggy_osds` won't be used any more, should we remove it too from the options ?

Reply:
Its still used in evict_cap_revoke_non_responders()

Reply:
Did I miss something here, I didn't find it:

```
Cscope tag: laggy_clients
   #   line  filename / context / line
   1    565  mds/Server.h <<GLOBAL>>
             std::set<client_t> laggy_clients;
   2    503  mds/Beacon.cc <<notify_health>>
             auto&& laggy_clients = mds->server->get_laggy_clients();
   3    504  mds/Beacon.cc <<notify_health>>
             if (!laggy_clients.empty()) {
   4    506  mds/Beacon.cc <<notify_health>>
             for (const auto& laggy_client: laggy_clients) {
   5    514  mds/Beacon.cc <<notify_health>>
             if (laggy_clients.size() <= (size_t)g_conf()->mds_health_summarize_threshold) {
   6    520  mds/Beacon.cc <<notify_health>>
             *css << "Many client (" << laggy_clients.size()
   7    523  mds/Beacon.cc <<notify_health>>
             m.metadata["client_count"] = stringify(laggy_clients.size());
   8    332  mds/Server.h <<get_laggy_clients>>
             return laggy_clients;
   9    335  mds/Server.h <<clear_laggy_clients>>
             laggy_clients.clear();
Type number and <Enter> (q or empty cancels): 

```